src/components/Menu/index.tsx

```diff
@@ -88,7 +88,8 @@ function Menu(props: MenuProps, ref) {
     (props.url.startsWith('/lib') || props.url.startsWith('/lib-v1')) &&
     (props.filterKey == 'ios' ||
       props.filterKey == 'android' ||
-      props.filterKey === 'flutter')
+      props.filterKey === 'flutter' ||
+      props.filterKey === 'js')
```

Comment:
need to add react-native to this list if we want the version switcher to appear for react-native docs

Reply:
This was left out intentionally as React Native is not in scope for the v6 Dev Preview (i.e. only JS should have the switcher)